extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzScheduler.java

```diff
@@ -230,6 +231,15 @@ public void pause() {
         }
     }
 
+    @Override
+    public void pause(String identity) {
+        try {
```

Comment:
We should probably add a null check, e.g. `java.util.Objects.requireNonNull(T)`..

Reply:
is added